app/_how-tos/configure-konnect-identity-upstream-oauth.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Configure the Upstream OAuth plugin with {{site.konnect_short_name}} Identity
+content_type: how_to
+breadcrumbs:
+  - /konnect-identity/
+
+tech_preview: true
+entities:
+  - route
+  - service
+  - plugin
+plugins:
+  - upstream-oauth
+products:
+    - gateway
+works_on:
+  - konnect
+tags:
+    - authentication
+description: Learn how to configure {{site.konnect_short_name}} Identity with Upstream OAuth.
+
+tldr: 
+  q: How do I configure {{site.konnect_short_name}} Identity with Upstream OAuth?
+  a: | 
+    Create a {{site.konnect_short_name}} Identity auth server, scope, claim, and client. Configure the OIDC plugin with your token endpoint, client ID, client secret, and scope. Generate a client token by sending a GET request to `$ISSUER_URL/oauth/token` and use the access token in a header when you send a request to the Gateway Service.
+
+tools:
+    # - konnect-api
+    - deck
+  
+prereqs:
```

Comment:
Since we are going public beta, Kong Identity will be removed from Labs. The API FF will be enabled for all customers. We can remove this pre-req.